SDPi_Supplement/asciidoc/volume2/gateways/tf2-ch-b-gateway-pv1-mapping.adoc

```diff
@@ -0,0 +1,132 @@
+===== PV1 - Patient Visit Segment
+The HL7 Patient Visit (PV1) segment requires a mapping from the SDC patient and location context information to the PV1 segment fields.
+
+.Prerequisite of Valid Patient & Location Context
+[NOTE#ref_valid_pt_loc_ctxt_note]
+====
+The SDC patient and location context information shall only be mapped to the corresponding fields in the HL7 PV1 segment when the requirements (e. g. inferred patient and/or location ensemble context, associated patient and/or location context, etc.) as defined in the <<ieee_11073_10700_2022>> are fulfilled.
+
+If the SDC patient and/or location context information shall not be used for the mapping, please refer to the <<ihe_pcd_tf-2_2019>> on how to populate the fields of the PV1 segment in this case.
+====
+
+====== PV1-2 Patient Class
+The PV1-2 field shall contain the code for the patient class. The *HL7 table 0004 - Patient Class* defines a set of recommended codes to be used for the data fields of the HL7 data type *IS* used in the PV1-2 field.
+
+Usually, a PoC device is used for patients admitted to a care unit in the hospital, and therefore, the field shall be set to *"I"* (Inpatient). If the patient class is unknown, the field shall be set to *"U"* (Unknown).
+
+The SDC data model does not support the concept of a patient class. Therefore, the field shall either be set to *"U"* (Unknown) by default, or set to a configurable value by the gateway.

```

Comment:
"BICEPS participant model"

Reply:
I referred to the "SDC MDIB". My understanding is that BICEPS also includes MDIB, Discovery, etc. which is a superset of what is described in the mapping. Would you agree?

Reply:
changed it to <<acronym_mdib>>.